pkg/v1/tkg/tkgpackageclient/package_update.go

```diff
@@ -117,6 +120,11 @@ func (p *pkgClient) updateValuesFile(o *tkgpackagedatamodel.PackageOptions, pkgI
 		}
 	}
 
+	pkgInstallToUpdate.Spec.Values = []kappipkg.PackageInstallValues{
```

Comment:
Bringing this logic into ```UpdatePackage``` will increase the ```gocyclo``` complexity as it will need another conditional check in ```UpdatePackage```.